crates/astria-sequencer/src/service/consensus.rs

```diff
@@ -151,23 +159,39 @@ impl Consensus {
         })
     }
 
-    #[instrument(skip(self))]
+    #[instrument(skip_all, fields(
+        height = %prepare_proposal.height,
+        tx_count = prepare_proposal.txs.len(),
+        time = %prepare_proposal.time
+    ))]
     async fn handle_prepare_proposal(
         &mut self,
         prepare_proposal: request::PrepareProposal,
     ) -> response::PrepareProposal {
         self.app.prepare_proposal(prepare_proposal).await
     }
 
-    #[instrument(skip(self))]
+    #[instrument(skip_all, fields(
```

Comment:
This might a be a few too many fields in the span. On the other hand these all seem useful. Not sure

Reply:
It's large but I think it's all useful if something goes wrong here, and it should all be relatively small amounts of data. A bunch of hashes.